### PR TITLE
REM-392

### PR DESCRIPTION
**Pull request by imtithalhalalo**
adding salesSettings page
page: 'sales-settings'

### CODE REVIEW

File: src/pages/sales-settings/Form/SalesSettingsForm.js [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+      sdpItemName: false,
+      sdpUnitPrice: false,
+      allowSalesNoLinesTrx: false,
+      salesTD: false
+    },
+    enableReinitialize: false,
+    maxAccess: access,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      maxReturnDays: yup.number().min(0, 'min').max(9999, 'max')
```
Comment (Alinadimmokdad) [inline src/pages/sales-settings/Form/SalesSettingsForm.js:55]: is it importnt to use min & max

Comment (imtithalhalalo) [reply src/pages/sales-settings/Form/SalesSettingsForm.js:55]: yes it is requirement

File: src/pages/sales-settings/Form/SalesSettingsForm.js [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+          }
+        });
+
+        const response = await postRequest({
+          extension: SystemRepository.Defaults.set,
+          record: JSON.stringify({ sysDefaults: data })
+        })
+
+        if (response) {
+          toast.success(platformLabels.Added)
```
Comment (RebeccaFrem02) [inline src/pages/sales-settings/Form/SalesSettingsForm.js:80]: shouldn't this be edited instead of added?

Comment (imtithalhalalo) [reply src/pages/sales-settings/Form/SalesSettingsForm.js:80]: Yes

**Review by RebeccaFrem02 — APPROVED** [review]

**Pull request merged**